src/elements/content-sidebar/activity-feed/annotation-thread/AnnotationThreadContent.js

```diff
@@ -54,6 +55,16 @@ const AnnotationThreadContent = ({
         errorCallback: onError,
     });
 
+    const onReplyEditHandler = (
+        id: string,
+        text: string,
+        status?: FeedItemStatus,
+        hasMention?: boolean,
```

Comment:
It looks like this value is unused. Can we omit it?

Reply:
I've actually needed to create this internal handler (and add `hasMention` argument) as in the end it's passed to a `Comment` component's `onEdit` prop, which is called with `hasMention` as 4th argument. `ThreadedReplies` api actually does not require `hasMention` so it's being abandoned here and not used in `handleEditReply` call.